packages/batch/src/constants.ts

```diff
@@ -1,8 +1,8 @@
-/**
```

Comment:
Curious why we remove the documentation here?

Reply:
I removed the comments at the top of the files as it's not a convention we use in this codebase. We tend to prefer adding comments to specific entities rather than on the file itself. This is something that [I pointed out](https://github.com/aws-powertools/powertools-lambda-typescript/pull/1588#discussion_r1258312831) during the review of your first PR for your utility.

The only exception to this are test files, in those files the header comment is used by [jest-runner-groups](https://www.npmjs.com/package/jest-runner-groups) to identify and run tests of the same group.